helm/helm.go

```diff
@@ -81,6 +81,10 @@ func main() {
 					Name:  "force, aye-aye",
 					Usage: "Perform install even if dependencies are unsatisfied.",
 				},
+				cli.BoolFlag{
+					Name:  "dry-run",
+					Usage: "Run but do not install into Kubernetes.",
```

Comment:
That would be good to display. We could just print the `kubectl` command on stdout when `uploadManifests` is called.

Does "perform the fetch, but only show kubectl commands" sound like a good docstring? Or is that not the intended behaviour we want?